### PR TITLE
Create Yu-Gi-Oh! Forbidden Memories (USA) (GameShark).cht

### PR DESCRIPTION
**Pull request by WojakFeelstone**
Cheats for Yu-Gi-Oh! Forbidden Memories (USA). I would appreciate if anyone else could test them.

### CODE REVIEW

**Comment by RobLoach** [comment]
Looks good! I haven'ted tested it, but feel free to and report back if it needs some updates.

Nicely done!

**Comment by WojakFeelstone** [comment]
Thanks! I just tested here again: everything seems to be working fine, with the exception of the "Skip Intro" and the "All Cards" cheat. Turns out the "All Cards" is divided into 19 parts, otherwise it may not work on most emulators. However, I don't know what happened with the "Skip Intro". I'll add the codes for the 19 sets of cards instead and see it that works.

File: cht/Sony - PlayStation/Yu-Gi-Oh! Forbidden Memories (USA) (GameShark).cht [hunk 1]
```diff
@@ -0,0 +1,33 @@
+cheats = 8
+
+cheat0_desc = "All Cards (1/4)"
+cheat0_code = "5000FF02+0000"
+cheat0_enable = "false"
+
+cheat1_desc = "All Cards (2/4)"
+cheat1_code = "801D0250+0505"
```
Comment (WojakFeelstone) [inline cht/Sony - PlayStation/Yu-Gi-Oh! Forbidden Memories (USA) (GameShark).cht]: cheat1_code = "801D0250+C8C8"

File: cht/Sony - PlayStation/Yu-Gi-Oh! Forbidden Memories (USA) (GameShark).cht [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+cheat1_enable = "false"
+
+cheat2_desc = "All Cards (3/4)"
+cheat2_code = "50006A02+0000"
+cheat2_enable = "false"
+
+cheat3_desc = "All Cards (4/4)"
+cheat3_code = "801D044E+0505"
```
Comment (WojakFeelstone) [inline cht/Sony - PlayStation/Yu-Gi-Oh! Forbidden Memories (USA) (GameShark).cht]: cheat3_code = "80105F96+C8C8"

**Pull request merged**